ground/gcs/src/plugins/scope/scopes3d/ffft/DynArray.h

```diff
@@ -0,0 +1,101 @@
+/*****************************************************************************
+
+        DynArray.h
+        By Laurent de Soras
+
+--- Legal stuff ---
+
+This program is free software. It comes without any warranty, to
+the extent permitted by applicable law. You can redistribute it
+and/or modify it under the terms of the Do What The Fuck You Want
+To Public License, Version 2, as published by Sam Hocevar. See
+http://sam.zoy.org/wtfpl/COPYING for more details.

```

Comment:
Is this license GPL compatible? If not, we could use [FFTW](http://www.fftw.org/)